miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SampleIdentityController.java

```diff
@@ -50,10 +50,6 @@
         log.error("Looking for sample categories", e);
       }
     }
-    if (results.isEmpty()) {
-      return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-    } else {
-      return new ResponseEntity<>(results, HttpStatus.OK);
-    }
+    return new ResponseEntity<>(results, HttpStatus.OK);
```

Comment:
No need to use ResponseEntity. Can just make the method return Set\<SampleCategoryDto\> (doesn't really matter though)